source/common/tcp_proxy/hash_policy.cc

```diff
@@ -0,0 +1,54 @@
+#include "common/tcp_proxy/hash_policy.h"
+#include "envoy/common/exception.h"
+
+namespace Envoy {
+namespace TcpProxy {
+
+class SourceIpHashMethod : public HashPolicyImpl::HashMethod {
```

Comment:
Whatever we do here can we move this to some common location so I can use it in the udp proxy I am working on. Somewhere in common/network is fine?

Reply:
Done